echidna/core/spectra.py

```diff
@@ -108,14 +113,19 @@ def sum(self):
         """
         return self._data.sum()
 
-    def normalise(self, count):
-        """ Normalise the total counts in the spectra to count, i.e. times each
-        bin by count / self.sum().
+    def scale(self, num_decays):
+        """ Scale THIS spectra to represent *num_decays* worth of decays.
+        
+        This rescales each bin by the ratio of *num_decays* to 
+        *self._num_decays*, i.e. it changes the spectra from representing
+        *self._num_decays* to *num_decays*. *self._num_decays* is updated
+        to equal *num_decays* after.
```

Comment:
Can you add in that if shrink method has been used on spectra, this will still scale with respect to the "un-shrunk" spectra